src/Module/Api/Twitter/Friends/Lists.php

```diff
@@ -92,6 +92,6 @@ protected function rawContent(array $request = [])
 
 		self::setLinkHeader();
 
-		System::jsonExit($return);
+		$this->response->exit('lists', ['lists' => $return]);
```

Comment:
```suggestion
		$this->response->exitWithJson($return);
```

Reply:
The Twitter endpoints should support XML.